lib/components/forms/NamespaceClaimForm.js

```diff
@@ -45,6 +50,15 @@ class NamespaceClaimForm extends React.Component {
           const cluster = this.props.clusters.items.find(c => c.metadata.name === values.cluster)
           const name = values.name
           const resourceName = `${cluster.metadata.name}-${name}`
```

Comment:
any chance the above might not find the cluster (causing cluster.metadata.name to blow up)?

Reply:
Yes there is potential for that, but I have a feeling it would break far sooner than here 😅
Maybe one for another time.